storage-bigtable/src/bigtable.rs

```diff
@@ -680,6 +680,29 @@ impl<F: FnMut(Request<()>) -> InterceptedRequestResult> BigTable<F> {
         deserialize_protobuf_or_bincode_cell_data(&row_data, table, key)
     }
 
+    pub async fn get_protobuf_or_bincode_cells<B, P>(
+        &mut self,
+        table: &str,
+        row_keys: Vec<RowKey>,
```

Comment:
It looks like the `Vec` input requirement can be avoided here:
```suggestion
        row_keys: &[RowKey],
```

Reply:
fixed in 963dc46d4ff0eface01a88e4559631b246052334